PROV/prov-cco-directmappings.ttl

```diff
@@ -42,10 +42,10 @@ prov:End                      owl:equivalentClass cco:ProcessEnding ;
 
 
 # Object Property mappings
-prov:wasEndedBy           rdfs:subPropertyOf cco:affects ;
-                          rdfs:comment "TODO add justification"@en .
-prov:wasStartedBy         rdfs:subPropertyOf cco:affects ;
-                          rdfs:comment "TODO add justification"@en .
+prov:wasEndedBy           rdfs:subPropertyOf cco:is_affected_by ;
+                          rdfs:comment "Ending an activity (which is a process) is a way that an entity (which is an Agent) can affect the activity"@en .
+prov:wasStartedBy         rdfs:subPropertyOf cco:is_affected_by ;
+                          rdfs:comment "Starting an activity (which is a process) is a way that an entity (which is an Agent) can affect the activity"@en .
```

Comment:
Seems like they are both is_affected_by with extra conditions: in one case, one of the two processes has first instant the same instant in which the starting activity takes place. In the other, one of the two processes has last instant the same instant in which the activity takes place. These are both CCO relations